pages/common/mongoexport.md

```diff
@@ -0,0 +1,32 @@
+# mongoexport
+
+> Produces a JSON or CSV export of data stored in a MongoDB instance.
```

Comment:
```suggestion
> Export data stored in a MongoDB instance in JSON or CSV format.
```

Reply:
I'm not really sure that this is that meuch clearer, except I do agree that it should be in the infinitive i.e. `produce`/`export` rather than `produces`/`exports`.

Reply:
I tried to use the command description from the man page: https://docs.mongodb.com/database-tools/mongoexport/.

> mongoexport is a command-line tool that produces a JSON or CSV export of data stored in a MongoDB instance.

Reply:
Is a MongoDB instance more, than the data it contains, e.g. would it be wrong to say
```suggestion
> Produce a JSON or CSV export of a MongoDB instance.
```

Reply:
I think it's probably more accurate to say `of the data stored in`:
```suggestion
> Produce exports of data stored in a MongoDB instance formatted as JSON or CSV.
```